server/services/healthCheck.test.ts

```diff
@@ -36,22 +36,18 @@ describe('Healthcheck', () => {
 
 function successfulCheck(name: string): HealthCheckService {
   return () =>
-    new Promise((resolve, _reject) => {
-      resolve({
-        name: `${name}`,
-        status: 'ok',
-        message: 'some message',
-      })
+    Promise.resolve({
```

Comment:
👌 

I see now, what I was talking about earlier doesn't apply here because this is an anonymous arrow function so this makes sense (if let's say this was an `async` function, you could return a value directly without having to wrap in `Promise.resolve()`, but this doesn't apply here)

Reply:
true, the whole thing could instead be

```typescript
async function successfulCheck(name: string): Promise<HealthCheckService> {
  return {
    name,
    status: 'ok',
    message: 'some message',
  }
}
```

but i don't want to diverge too much from what was in the template repo.